app/models/distribution.rb

```diff
@@ -125,6 +127,14 @@ def combine_distribution
     line_items.combine!
   end
 
+  def notify_partner!
+    self.scheduled!
+  end
+
+  def picked_up!
+    self.complete!
+  end
```

Comment:
I understand what you're doing here (providing a more read-friendly API), and I like that kind of thinking :+1:

I think in this case, since all code rots over time, and these methods are just basically aliasing existing methods, let's just stick with using the built-in enum magic methods and strike these ones out. The main concern I have there is that expanding the model's API creates more code to maintain.